label_studio/tasks/models.py

```diff
@@ -1001,6 +1001,50 @@ class Meta:
         db_table = 'prediction'
 
 
+class FailedPrediction(models.Model):
+    """
+    Class for storing failed prediction(s) for a task
+    """
+    message = models.TextField(
+        _('message'),
+        default='',
```

Comment:
When adding columns to existing models, it's not safe to use defaults at all per the migration guide; in this case I don't think we have a problem, but I would prefer we didn't mix null and empty string defaults for what are essentially string fields

Reply:
```suggestion
        default=None,
```